compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -882,8 +877,6 @@ impl<'a> Parser<'a> {
 
     /// Parses inline const expressions.
     fn parse_const_expr(&mut self, span: Span) -> PResult<'a, P<Expr>> {
-        self.sess.gated_spans.gate(sym::inline_const, span);
```

Comment:
If you remove this we will be missing the note when using the feature without the feature gate.

Reply:
Oh, thank you! I meant to ask about that part: for some reason I thought maybe using `eat_keyword` elsewhere meant I didn’t need this. Not sure why I thought that :)